projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -953,9 +953,51 @@ class='jetpack-form-file-field'
 		$field .= "<div class='jetpack-form-file-field__preview-wrap'></div>\n";
 		$field .= "</div>\n";
 
+		$this->localize_file_field_script();
+
 		return $field;
 	}
 
+	/**
+	 * Helper functions that localizes the jetpack-form-file-field script once.
+	 */
+	private function localize_file_field_script() {
+		/**
+		 * Filter to disable the localization of the file field script.
```

Comment:
What's the use case of this filter?

Reply:
It is there so that we can supress the localization from being added multiple times if the form is field is being includede more then once.

Reply:
I will try using wp_add_inline_script instead to see if this helps.

Reply:
Nope that this didn't work well :(